stoservice/src/main/java/com/tutrit/stoservice/controller/HelpController.java

```diff
@@ -1,12 +1,16 @@
 package com.tutrit.stoservice.controller;
 
+import java.security.PrivateKey;
 import java.util.Arrays;
 
 public class HelpController implements CommandController {
     private static final Command command = Command.HELP;
+    private final String ANY = "Any wrong input will return to the beginning";
 
     public void doCommand(Request request, Response response) {
-        response.setResponse(String.format(Arrays.toString(Command.values()), request.getCommand()));
+    response.setResponse(String.format(Arrays.toString(Arrays.stream(Command.values())
```

Comment:
What makes the String.format?

Reply:
A convenient way to put an array into a string.
The method is not mine, I just rewrote it in the stream.